src/hapi/pipelines/database/metadata.py

```diff
@@ -56,7 +56,7 @@ def populate(self):
                 resource_row = DBResource(
                     hdx_id=resource_id,
                     dataset_ref=dataset_row.id,
-                    filename=resource["filename"],
+                    name=resource["name"],
```

Comment:
This should be `name=resource["filename"]`, unless we also change it in the scraper (perhaps we should)

Reply:
Good catch!  I think it would be cleanest if we did, but since it's not at the core of the issue I think we can revisit when Mike is back.

Reply:
Sounds good, I've created [HAPI-246](https://humanitarian.atlassian.net/browse/HAPI-246?atlOrigin=eyJpIjoiYmNkY2IyNGJjZjkzNDEwMzhiM2IyODNhZTIxNWU1YmIiLCJwIjoiaiJ9), could you add a TODO referencing it in the places where it should be changed?